Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx

```diff
@@ -242,35 +242,35 @@
     <comment>Text for Hour</comment>
   </data>
   <data name="HoursIntervalTypeReminder" xml:space="preserve">
-    <value>Heures</value>
+    <value>Intervalle d'heures pour le rappel</value>
```

Comment:
@Kerubananthan3761 , Can re-check and confirm this?

Reply:
Checked, getting same text as **Intervalle d'heures pour le rappel**